wg-charter.html

```diff
@@ -0,0 +1,498 @@
+            strategies for the API, notably to address fingerprinting issues
+            and to prevent unauthorized use of computational resources.
+        </p>
+
+        <p>
+            Note that the majority of the input to this Working Group will come
+            directly from the <a href="https://www.w3.org/community/gpu/">GPU for the Web Community Group</a>.
+            No major development will happen in the Working Group itself. Instead, the Community
+            Group will be driving the technical work.
+        </p>
+
+        <h2>Out of Scope</h2>
+        <p>
+            The scope of work is restricted to the development of a programming
+            interface between the Web Platform and modern 3D graphics and
+            computation capabilities present on native system platforms. The work
+            will not define hardware features or algorithms, and the interface it
+            defines is not intended to be directly exposed by a GPU driver.
+        </p>
+
+      </section>
+
+
+
+      <section id="deliverables">
+        <h2>
+          Deliverables
+        </h2>
+
+        <p>Updated document status is available on the <a href="https://www.w3.org/groups/wg/gpu/publications">group publication status page</a>.</p>
+
+        <p><i>Draft state</i> indicates the state of the deliverable at the time of the charter approval. <i>Expected completion</i> indicates when the deliverable is projected to become a Recommendation, or otherwise reach a stable state.</p>
+
+        <div id="normative">
+          <h3>
+            Normative Specifications
+          </h3>
+          <dl>
+          <dt><a href="https://www.w3.org/TR/webgpu/">WebGPU API</a></dt>
+
+          <dd>
+              An API for performing operations, such as rendering and
+              computation, on a Graphics Processing Unit (GPU).
+              <p class="draft-status"><b>Draft state:</b> Working Draft</p>
+              <p><b>Adopted Draft:</b> <a href="https://www.w3.org/TR/2022/WD-webgpu-20220831/">31 August 2022</a></p>
+              <p><b>Exclusion Draft:</b> <a href="https://www.w3.org/TR/2021/WD-webgpu-20210518/">WebGPU 18 May 2021</a>;
+                <br/>associated <a href="https://www.w3.org/mid/cfe-8036-f5ffade17337af94e77d0c1e588bbe63549638fb@w3.org">Call for Exclusion</a> on 2021-05-18 ended on 2021-10-15;
+                <br/>produced under Working Group charter: <a href="https://www.w3.org/2020/12/gpu-wg-charter.html">https://www.w3.org/2020/12/gpu-wg-charter.html</a></p>
+              <p class="milestone"><b>Expected completion:</b> Q4 2024</p>
+          </dd>
+
+          <dt><a href="https://www.w3.org/TR/WGSL/">WebGPU Shading Language</a></dt>
+
+          <dd>
+              A Shading Language specification that defines the programmable
+              interface to the GPU for the Web graphics and computation
+              pipeline, and that can be translated or compiled into
+              platform-specific instructions.
```

Comment:
A possible addition might be similar to the API side we make an explicit point about representability in the shading languages of modern APIs. This currently could be read as WGSL will be compiled directly to machine code, but I might be trying to view it too naively for the purpose of reviewing.

(copied from https://github.com/gpuweb/admin/pull/19)